base_layer/core/tests/node_state_machine.rs

```diff
@@ -155,7 +155,7 @@ fn test_event_channel() {
         LivenessConfig {
             enable_auto_join: false,
             enable_auto_stored_message_request: false,
-            auto_ping_interval: Some(Duration::from_millis(100)),
+            auto_ping_interval: Some(Duration::from_millis(10)),
```

Comment:
Is this perhaps nog too quick? It might flood the network with liveness request?

Reply:
That's a test